### PR TITLE
Bug fixes for Issue #818

### PR DESCRIPTION
**Pull request by BolunDai0216**
# Description

- Changed the action space to `[-1, 1]`. Internally the action space is `[-1, 1] * purser_max_accel`.
- Changed the default max velocity and max acceleration of the pursuer to `[-0.2, -0.2]` and `[-0.02, 0.02]`
- Changed color of food and poison.
- Made the mass of food smaller by changing its density to 0.01

Fixes # 818

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [x] I have run `pytest -v` and no errors are present.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I solved any possible warnings that `pytest -v` has generated that are related to my code to the best of my knowledge.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: pettingzoo/sisl/waterworld/waterworld_models.py [hunk 1]
```diff
@@ -135,8 +136,8 @@ def __init__(
     @property
     def observation_space(self):
         return spaces.Box(
-            low=np.float32(-2 * np.sqrt(2)),
-            high=np.float32(2 * np.sqrt(2)),
+            low=np.float32(-4 * np.sqrt(2)),
+            high=np.float32(4 * np.sqrt(2)),
```
Comment (jjshoots) [inline pettingzoo/sisl/waterworld/waterworld_models.py:140]: Just curious, what are these magic numbers?

Comment (BolunDai0216) [reply pettingzoo/sisl/waterworld/waterworld_models.py]: They should be $\pm\sqrt{2}$, the max speed is defined for $v_{x}$ and $v_y$, so the max magnitude for the speed $v = [v_x, v_y]$ would be $\sqrt{v_x^2 + v_y^2} = \sqrt{2}v_\mathrm{max}$

**Review by jjshoots — APPROVED** [review]
LGTM!

**Comment by WillDudley** [comment]
lgtm :) thanks for this

**Pull request merged**